docs/concepts/rolling-upgrades.md

```diff
@@ -0,0 +1,23 @@
+---
+linkTitle: "Rolling Upgrades to Orchestrator"
+---
+
+# Upgrade Guard 
+
+You can use the rolling upgrade feature to upgrade an orchestrator (for example, Kubernetes or OpenShift) without causing service downtime/cluster failure. For example, OpenShift provides a one-click upgrade but that feature doesn't take into consideration Stateful workloads. 
+
+We have developed a separate component called the Upgrade guard. This component blocks certain nodes from being upgraded or drained thus avoiding data loss in the cluster.
+To use the rolling upgrade feature, you need to enable both the Node manager and the Upgrade Guard components (this is set on the `storageoscluster` CR).
+
+The upgrade guard will detect if a volume is unhealthy (for example, one that doesn't have enough synced replicas), at which point one or more node manager pods will become unavailable. Ondat uses the PodDisruptionBudget (PDB) to stop more than 1 node manager pod being unavailable at any point in time. 
```

Comment:
```suggestion
The upgrade guard will detect if a volume is unhealthy (for example, one that doesn't have enough synced replicas), at which point one or more node manager pods will become unavailable. Ondat uses a PodDisruptionBudget (PDB) to stop more than 1 node manager pod being unavailable at any point in time. 
```

This is a bit of a nit, but I think **a** PDB makes more sense here, as it's something we create (and there can be multiple of)